libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -247,12 +247,16 @@ void AP_Baro::calibrate(bool save)
     _guessed_ground_temperature = get_external_temperature();
 
     // panic if all sensors are not calibrated
+    uint8_t num_calibrated = 0;
     for (uint8_t i=0; i<_num_sensors; i++) {
         if (sensors[i].calibrated) {
-            gcs().send_text(MAV_SEVERITY_INFO, "Barometer calibration complete");
-            return;
+            gcs().send_text(MAV_SEVERITY_INFO, "Barometer[%u] calibration complete", i+1);
```

Comment:
Could this be "Barometer %u calibration complete" instead? The array index has historically meant the 0 index'ed count rather then what the user thinks of as first baro. This showed up in the airspeed, and it was also a poorer sentence then just having it written out.